volume/local/local_unix.go

```diff
@@ -23,17 +24,19 @@ var (
 		"type":   true, // specify the filesystem type for mount, e.g. nfs
 		"o":      true, // generic mount options
 		"device": true, // device to mount from
+		"helper": true, // mount helper, either built-in (by default) or external
 	}
 )
 
 type optsConfig struct {
 	MountType   string
 	MountOpts   string
 	MountDevice string
+	MountHelper string
 }
 
 func (o *optsConfig) String() string {
-	return fmt.Sprintf("type='%s' device='%s' o='%s'", o.MountType, o.MountDevice, o.MountOpts)
+	return fmt.Sprintf("type='%s' device='%s' o='%s' helper='%s'", o.MountType, o.MountDevice, o.MountOpts, o.MountHelper)
```

Comment:
Just fix it. Does this format land anywhere? Single quoting like this won't properly escape anything. If this gets consumed, `helper` can just escape out by ending the quote and inject anything.

If this is just for display, its just cleaner to use `%q`.

Please leave the code base cleaner than you found it.